### PR TITLE
feat: initial version

### PR DESCRIPTION
**Pull request by mhdawson**
## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [X] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [X] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [X] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by mhdawson** [comment]
What this needs before it can work/land is

1. Publishes of the ubi-base-stack which should occur one this PR lands - https://github.com/paketo-community/ubi-

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @paketo-buildpacks/builders-maintainers
```
Comment (mhdawson) [inline .github/CODEOWNERS:1]:
```suggestion
* @paketo-community/builders-maintainers
```

File: .github/workflows/push-image.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - name: Push to GCR
+      env:
+        GCR_PUSH_BOT_JSON_KEY: ${{ secrets.GCR_PUSH_BOT_JSON_KEY }}
+      run: |
+        echo "${GCR_PUSH_BOT_JSON_KEY}" | docker login --username _json_key --password-stdin gcr.io
+        docker tag builder "gcr.io/paketo-buildpacks/builder:buildpackless-base"
+        docker tag builder "gcr.io/paketo-buildpacks/builder:${{ steps.event.outputs.tag }}-buildpackless-base"
+
+        docker push "gcr.io/paketo-buildpacks/builder:buildpackless-base"
+        docker push "gcr.io/paketo-buildpacks/builder:${{ steps.event.outputs.tag }}-buildpackless-base"
```
Comment (mhdawson) [inline .github/workflows/push-image.yml:47]:
```suggestion
        docker push "gcr.io/paketo-community/builder:buildpackless-base"
        docker push "gcr.io/paketo-community/builder:${{ steps.event.outputs.tag }}-buildpackless-base"
```

File: .github/workflows/push-image.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      env:
+        PAKETO_BUILDPACKS_DOCKERHUB_USERNAME: ${{ secrets.PAKETO_BUILDPACKS_DOCKERHUB_USERNAME }}
+        PAKETO_BUILDPACKS_DOCKERHUB_PASSWORD: ${{ secrets.PAKETO_BUILDPACKS_DOCKERHUB_PASSWORD }}
+      run: |
+        echo "${PAKETO_BUILDPACKS_DOCKERHUB_PASSWORD}" | docker login --username "${PAKETO_BUILDPACKS_DOCKERHUB_USERNAME}" --password-stdin
+        docker tag builder "paketobuildpacks/builder:buildpackless-base"
+        docker tag builder "paketobuildpacks/builder:${{ steps.event.outputs.tag }}-buildpackless-base"
+
+        docker push "paketobuildpacks/builder:buildpackless-base"
+        docker push "paketobuildpacks/builder:${{ steps.event.outputs.tag }}-buildpackless-base"
```
Comment (mhdawson) [inline .github/workflows/push-image.yml:59]:
```suggestion
        docker tag builder "paketocommunity/builder:buildpackless-base"
        docker tag builder "paketocommunity/builder:${{ steps.event.outputs.tag }}-buildpackless-base"
        docker push "paketocommunity/builder:buildpackless-base"
        docker push "paketocommunity/builder:${{ steps.event.outputs.tag }}-buildpackless-base"
```

**Comment by mhdawson** [comment]
The base ubi stack has now landed so this should be ready to do and is the next step in getting the ubi components in place.

**Comment by mhdawson** [comment]
@builders-maintainers I'm hoping we could get a review of this as its  the next step to unlock future work on the ubi stack.

**Comment by mhdawson** [comment]
@paketo-community/builders-maintainers when you look at this one, it might also make sense to look at https://github.com/paketo-community/builder-ubi-base at the same time as I based both of those on the same base with the second one just adding the ubi-nodejs-extension for now.

File: builder.toml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+description = "ubi 8 base image with no buildpacks included. To use, specify buildpacks at build time."
+
+[lifecycle]
+  version = "0.16.4"
+
+[stack]
+  build-image = "docker.io/paketocommunity/build-ubi-base/build:0.0.1"
```
Comment (mhdawson) [inline builder.toml:7]:
```suggestion
  build-image = "docker.io/paketocommunity/build-ubi-base:0.0.1"
```

File: builder.toml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+description = "ubi 8 base image with no buildpacks included. To use, specify buildpacks at build time."
+
+[lifecycle]
+  version = "0.16.4"
```
Comment (mhdawson) [inline builder.toml:4]:
```suggestion
  version = "0.17.0-pre.2"
```

**Review by ForestEckhardt — APPROVED** [review]

**Pull request merged**